libsignal-service-actix/src/websocket.rs

```diff
@@ -38,19 +40,31 @@ impl From<AwcWebSocketError> for ServiceError {
                     reason: e.to_string(),
                 },
             },
+            AwcWebSocketError::ProtocolError(e) => match e {
+                WsProtocolError::Io(e) => match e.kind() {
+                    std::io::ErrorKind::UnexpectedEof => {
+                        ServiceError::WsClosing {
+                            reason: format!(
+                                "IO error during WebSocket connection: {}",
```

Comment:
maybe have a slightly different message, like "closing websocket connection because: {}" so it's easier to differentiate compared to the error line 55?